sbt-scalajs-bundler/src/main/scala/scalajsbundler/PackageJson.scala

```diff
@@ -51,6 +51,9 @@ object PackageJson {
       JSON.obj(
         (
           additionalNpmConfig.toSeq :+
+          "private" -> JSON.bool(true) :+
+          "license" -> JSON.str("UNLICENSED") :+
+          "description" -> JSON.str("") :+
```

Comment:
Wow, the scope of this PR is getting wider and wider ;)

I’m not sure we should force these fields to these values. An alternative solution would be to put these fields in the default value of `additionlNpmConfig` so that users can still change them if they want to.